api/openapi/v2/core-metadata.yaml

```diff
@@ -361,8 +356,47 @@ components:
         - $ref: '#/components/schemas/BaseResponse'
       type: object
       properties:
-        device:
-          $ref: '#/components/schemas/Device'
+        id:
```

Comment:
Still use $ref: '#/components/schemas/Device' for `properties` rather than device

Reply:
Ok, feel we may be going in circles here... ;-) But the code now references dto.Device so this should be put back to:

```
        device:
          $ref: '#/components/schemas/Device'
```

Make this same change for all the Response DTO's so they match the code.